MokManager.c

```diff
@@ -1887,15 +1895,16 @@ static EFI_STATUS mok_hash_enroll(void)
 	EFI_FILE *file = NULL;
 	UINTN filesize;
 	void *data;
+	CHAR16 *selections[] = {
+		L"Select Binary",
+		L"",
+		L"The Selected Binary will have its hash Enrolled",
+		L"This means it will Subsequently Boot with no prompting",
+		L"Remember to make sure it is a genuine binary before Enroling its hash",
```

Comment:
I learned both 'enrol' and 'enroll' are valid; but we should use the same spelling in both uses in this string.